### ISSUE TITLE
Experiment with dropping layers.

### ISSUE THREAD

**Issue by itsShnik** [itsShnik/adaptively-finetuning-transformers#27]
[LayerDrop](https://arxiv.org/pdf/1909.11556.pdf) presents a way to train a transformer model resistant to varying depths using structured dropout, which allows pruning layers at test time. However, it does not propose a method to find out which layers to drop and which to not. Can we do that using a policy? Can it be done only while fine-tuning?
